net/golioth/lightdb.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2021-2022 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <net/golioth.h>
+
+#include "coap_utils.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(lightdb, CONFIG_GOLIOTH_LOG_LEVEL);
+
+int golioth_lightdb_get(struct golioth_client *client, const uint8_t *path,
```

Comment:
Does it make sense to declare these functions in a new file `lightdb.h`? Currently, they are defined in `golioth.h`.

Reply:
Makes sense, it will be more consistent with rpc and settings services. Also it seems easier to find lightdb related APIs, as they are group in one file.

Updated!